ui/shared/pagination/useQueryWithPages.ts

```diff
@@ -41,6 +41,17 @@ function getNextPageParams<R extends PaginatedResources>(data: ResourcePayload<R
     return;
   }
 
+  const nextPageParams = data.next_page_params;
+  if (nextPageParams) {
+    // check if a page param is "null" and remove it
+
+    return Object.entries(nextPageParams)
+    //eslint-disable-next-line @typescript-eslint/no-unused-vars
+      .filter(([ _, v ]) => v != null)
+      .reduce((acc, [ k, v ]) => ({ ...acc, [k]: v }), {});
+
+  }
```

Comment:
Can you explain logic behind this? Why should it be removed? The `null` is a valid value of `next_page_params` property for all of our API response type

Reply:
This code is neccesary because the endpoint thats being use in Account History can return some null fields that are important for getting the next page. If we remove it, the second page can fail.

Reply:
Yeah so essentially to have pagination working for `noves_address_history` this appears to be necessary at the moment.

Reply:
> This code is neccesary because the endpoint thats being use in Account History can return some null fields that are important for getting the next page

As I understand the code, it removes from `next_page_params` object all fields with `null` value. As you said, why are they being removed if they are important? Nevertheless, this change will break integration with our API since it relies on all fields being present in the request for the next page, even on those whose value is `null`.

Reply:
My guess is if the parameter is not needed for the next page request, it should be omitted from `next_page_params`.

Reply:
We changed how our backend works to accommodate this. The code mentioned in this convo has been removed.